src/containers/search/SearchCard.tsx

```diff
@@ -117,98 +122,115 @@ export const SearchCard = forwardRef<HTMLDivElement, SearchCardProps>(
               media={facility.media}
             />
           </Stack>
-          <Stack
-            // fill={true}
-            onClick={() => navigate(`/facility/${facility.id}`)}
-            justifyContent="space-between"
-            width="100%"
-            // minWidth={theme.spacing(27)}
-            spacing={0.5}
-            sx={{ p: 1, mt: 0, cursor: 'pointer' }}
+          <Link
```

Comment:
This opens in a new tab right?

Reply:
yes, but  i will change this in a coming pr. Right now (also on test) a refresh happens if you go back from facility detail to search page. This is a short-term workaround to create a better experience